roles/common/molecule/default/verify.yml

```diff
@@ -4,7 +4,7 @@
   gather_facts: false
   tasks:
   - name: check common package status
-    package:
+    ansible.builtin.package:
```

Comment:
can we use `ansible.builtin.apt` instead.

Reply:
Done @kayiwa!